stable/csi-secrets-store-provider-aws/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: csi-secrets-store-provider-aws
-version: 0.0.1
+version: 0.0.3
```

Comment:
Why 0.0.3? What about 0.0.2?

Reply:
0.0.2 was the first commit, but can change it to 0.0.2 for both.

Reply:
I vote for 0.0.2

Reply:
The version of the chart must be incremented over the version on the master, not in every PR commit.

Reply:
Fixed.